AzureSkies/AzureSkies/Controllers/FlightsController.cs

```diff
@@ -23,84 +35,57 @@ public class FlightsController : ControllerBase
         // DI
         public FlightsController(AzureSkiesDbContext context, IFlightStatus service)
         {
-            _context = context;
             _service = service;
         }
 
```

Comment:
Having these controller methods give something back from the service could be valuable for debug/building additional architecture.